docs/understanding-the-flutterflow-ui/Projects/how-to-create-find-organize-projects.md

```diff
@@ -0,0 +1,30 @@
+---
+slug: how-to-create-find-organize-projects
+title: How to Create, Find, and Organize Projects
+tags: []
+toc_max_heading_level: 5
+---
+## How to Create a Project
+To create a new project, go to the Dashboard and click the “Create New” button in the upper right corner. This will open the “Create Project” window where you can choose to start with a template app or a blank project. Name the project and then select a blank project or template. 
+
+During project creation, you can access a three-step "Project Setup" window, which is optional. If you prefer to bypass this, simply click "Skip." All setup options will remain accessible under “Settings and Integrations” → “Firebase” → Wizard hat icon.
+
+## How to Find Projects
+Navigate to the Project Dashboard to view all your projects. You can search for specific projects using the search bar. Narrow your search scope with the dropdown menu next to the search bar:
+
+- My Private Projects: Projects accessible only to you.
+- My Shared Projects: Projects you own and have shared with others.
+- Shared With Me: Projects shared with you that you do not own.
+
+## **Organizing Projects**
+### **Creating and Managing Tags**
+To create a tag, click the orange “Tag” button or access it via the three dots in the upper right corner of a project panel, then select “Add Tag”. Tags help categorize and filter your projects for easier management.
+
+### **Adding Tags to Projects**
+Add a tag to a project by clicking the three dots in the upper right corner of the project panel and selecting a tag. Each project can have only one tag.
+
```

Comment:
Maybe can we add a visual asset for tags?

Reply:
Probably a screenshot of how to add one would be better

Reply:
added